structs.go

```diff
@@ -112,6 +112,37 @@ type Session struct {
 	wsMutex sync.Mutex
 }
 
+// Connection is a Connection returned from the UserConnections endpoint
+type Connection struct {
```

Comment:
Connection is pretty generic, could we call this UserConnection

Reply:
GuildConnection and GuildAccount share the same fields, would it be possible to rename them to use the same structs

Reply:
Guh, don't really want to break the API, would be happy embedding:

type GuildIntegration struct {
  Integration
}